src/metabase/task/send_anonymous_stats.clj

```diff
@@ -0,0 +1,44 @@
+(ns metabase.task.send-anonymous-stats
+  "Contains a Metabase task which periodically sends anonymous usage information to the Metabase team."
+  (:require [clojure.tools.logging :as log]
+            (clojurewerkz.quartzite [jobs :as jobs]
+                                    [triggers :as triggers])
+            [clojurewerkz.quartzite.schedule.cron :as cron]
```

Comment:
I'll note that every other task has this same issue ;)
Also, I couldn't figure out the syntax for putting a double namespace  in the above grouping.